freqtrade/rpc/api_server/api_v1.py

```diff
@@ -508,3 +509,19 @@ def sysinfo():
 @router.get("/health", response_model=Health, tags=["info"])
 def health(rpc: RPC = Depends(get_rpc)):
     return rpc.health()
+
+
+@router.get("/trades/open/custom-data", response_model=list[ListCustomData], tags=["info"])
```

Comment:
Tags for these endpoints shouldn't be "info".

they should (from a location point of view) be moved near the other "/trades/" calls.